docs/README-EDITING.md

```diff
@@ -66,77 +46,128 @@ sitemap:
+### JSDoc links
 
-### Editing the `Next` version
+As we want to support case-insensitive URLs, follow these guidelines when editing the JSDoc links:
 
-*The next version is only available on the `localhost` and on the staging server.*
+* Use lower case characters.
+* To separate words before `#`, use `-`.
 
-Files included in next versions live in dir `/next/`.
-To display it in a browser, please go into URL: `/docs/next`.
+```js
+// Wrong:
+/** {@link Options#autoColumnSize}. */
 
-### Editing the `latest` version
+// OK: 
+/** {@link options#autocolumnsize Options#autoColumnSize}. */
 
-The latest version is the largest semver valid directory inside the root directory.
-It is in `/[number]/`.
 
-To display the latest docs version in a browser, please go into: `/docs/`.
+// Wrong:
+/** {@link HiddenRows} */
 
-### Editing an archival version
+// OK:
+/** {@link hidden-rows HiddenRows} */
+```
 
-Directory: `/[number]/`.
+## Documentation versioning
 
-In a browser please go into: `/docs/[number]`.
+To create a new version of the Handsontable documentation:
 
-## Editing API Reference
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
+```
 
-To edit API Reference goes into source code and change jsdocs comments. Then, to generate API Reference pages:
+To remove an existing version of the Handsontable documentation:
 
-```shell script
-npm run docs:api
+* Remove the required version's [directory](./README.md#handsontable-docs-directory-structure):
+```bash
+rm -rf ./<semver.version>
 ```
 
-## Versioning
+## Markdown containers
+
+To render content in different ways, the documentation uses custom Markdown containers, for example:
+
+```markdown
+::: example #exampleId .class :preset --html 1 --js 2
 
-**Release a version**
+// code example
 
-```shell script
-npm run docs:version
+:::
 ```
 
-Then you'll be asked for the version that you want to release (interactive mode).
+We use the following Markdown containers:
 
-**Remove a version**
+| Container                    | Usage                                                      |
+|------------------------------|------------------------------------------------------------|
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
 
-It is enough to remove version directory.
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that show the code's result:
 
-```shell script
-rm -rf ./[NUMBER]
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+
+    ```js
+    // code here
+    ```
+:::
 ```
```

Comment:
Thank you for adding this example of how to insert code preview. I think it is fair to add two snippets more inside (HTML, CSS):

```js
::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
// `#example1` is the example's ID, for creating a Handsontable container
    ```html
    <div id="example1"></div>
    ```
    ```js
    // code here
    ```
    ```css
    /* custom css here */
    ```
:::
```